src/api.v2/events/handleWorkRequest.js

```diff
@@ -26,7 +47,7 @@ const handleWorkRequest = async (socket, data) => {
     logger.log(`[REQ ??, SOCKET ${socket.id}] Error while processing WorkRequest event.`);
     logger.trace(e);
 
-    socket.emit(`WorkResponse-${uuid}`, {
+    socket.emit(`WorkResponse-${ETag}`, {
```

Comment:
`uuid` isn't a port that is being heard by the UI as far as I know, I think this was a bug

Reply:
what would be the effect of this being a bug? what should be broken

Reply:
This message wasn't reaching the UI